src/libraries/illink-oob.targets

```diff
@@ -24,7 +25,7 @@
       <OOBAssemblyToIgnore Include="System.Speech" />
 
       <NetCoreAppCurrentAssembly Include="$(NetCoreAppCurrentRuntimePath)*.dll"
-                                 Exclude="$(NetCoreAppCurrentRuntimePath)*.Generator.dll;
+                                 Exclude="$(NetCoreAppCurrentRuntimePath)*.Generator*.dll;
```

Comment:
why is this needed?

Reply:
The generator assembly namens aren't consistent in their naming. There was at least one generator that wasn't excluded. I believe it was CoreLib's.

Reply:
ah I see there's `System.Private.CoreLib.Generators.dll` indeed

Reply:
I'm surprised we're placing generators in the runtime path, that's wrong.  They don't even target the same thing.  We should see about fixing that (separately is fine).